src/main/kotlin/racingCar/Application.kt

```diff
@@ -1,9 +1,14 @@
 package racingCar
 
+import racingCar.controller.RacingGame
+import racingCar.model.Cars
+import racingCar.view.InputView
+import racingCar.view.OutputView
+
 fun main() {
-    val players = InputView.inputCarNames().split(",")
+    val carName = InputView.inputCarNames().split(",")
```

Comment:
`List<String>` 이니까 아래와 같은 이름은 어떨까요?

```suggestion
    val carNames = InputView.inputCarNames().split(",")
```